rust/auction-house/src/lib.rs

```diff
@@ -1345,11 +1345,12 @@ pub const AUCTION_HOUSE_SIZE: usize = 8 + //key
 32 + //treasury mint
 32 + //authority
 32 + // creator
-8 + // bump
-8 + // treasury_bump
-8 + // fee_payer_bump
+1 + // bump
+1 + // treasury_bump
+1 + // fee_payer_bump
```

Comment:
good catch, but to keep this all the same size we need to move the extra 21 bytes to the padding.

Reply:
Addressed by 6affeef0d24b158f062e389ed00c5ad9c47aa83b